e2e/core/viewer/viewer-component.e2e.ts

```diff
@@ -452,4 +455,45 @@ xdescribe('Viewer', () => {
             });
         });
     });
+
+    describe('Code viewer', () => {
+
+        let jsFileInfo = new FileModel({
+            'name': resources.Files.ADF_DOCUMENTS.JS.file_name,
+            'location': resources.Files.ADF_DOCUMENTS.JS.file_location
+        });
+
+        let jsFileUploaded;
+
+        beforeAll(async (done) => {
+            await this.alfrescoJsApi.login(acsUser.id, acsUser.password);
+
+            jsFileUploaded = await uploadActions.uploadFile(this.alfrescoJsApi, jsFileInfo.location, jsFileInfo.name, '-my-');
+
+            loginPage.loginToContentServicesUsingUserModel(acsUser);
+
+            done();
+        });
+
+        afterAll(async (done) => {
+            await this.alfrescoJsApi.login(acsUser.id, acsUser.password);
+            await uploadActions.deleteFilesOrFolder(this.alfrescoJsApi, jsFileUploaded.entry.id);
+            done();
+        });
+
+        it('[C297698] Should display code editor when opening .js file', () => {
```

Comment:
the name of this describe should be something that highlight the thing that we are testing an extension